hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/TokenAirdropHandlerTest.java

```diff
@@ -462,6 +467,62 @@ void handleAirdropMultipleTokensTransfers() {
         assertThat(relationToFungible.balance()).isEqualTo(1000L);
     }
 
+    @Test
+    void handleAirdropWithCustomFees() {
+        // setup all states
+        handlerTestBaseInternalSetUp(true);
+
+        // setup airdrop states
+        givenAssociatedReceiver(tokenReceiverId, nonFungibleTokenId);
+        givenAssociatedReceiver(tokenReceiverId, fungibleTokenId);
+        refreshReadableStores();
+        refreshWritableStores();
+        givenStoresAndConfig(handleContext);
+        given(handleContext.dispatchMetadata()).willReturn(DispatchMetadata.EMPTY_METADATA);
+        // mock record builder
+        tokenAirdropHandler = new TokenAirdropHandler(tokenAirdropValidator, validator);
+
+        final var fees = List.of(
+                withFixedFee(FixedFee.newBuilder().amount(66).build())
+                //                ,withFractionalFee(fractionalFee.copyBuilder().netOfTransfers(true).build())
```

Comment:
The commented out code in these tests were here before, right?

Reply:
oh, sorry. I wasn't actually ready for review yet. I created the PR so I could ask Neeha some questions. I'll delete those comments.